src/plugins/vis_type_vega/public/vega_inspector/vega_inspector.scss

```diff
@@ -0,0 +1,13 @@
+.insVegaViewWrapper {
+  height: 100%;
+  > div {
+    height: 100%;
+  }
```

Comment:
Can you add the vega vis folder to the .sass-lint.yml? It should fix the formatting in this file

Reply:
done